src/ace_header.F90

```diff
@@ -110,6 +111,10 @@ module ace_header
     integer                :: urr_inelastic
     type(UrrData), pointer :: urr_data => null()
 
+    ! Multipole data
+    logical                           :: mp_present
+    type(MultipoleArray), allocatable :: multipole
```

Comment:
If you revert this back to being a `pointer` rather than an allocatable scalar, it gets rid of the weird segfault with gfortran 5.2.1. Seems like gfortran's support for allocatable scalars is still spotty, but I haven't been able to reduce the problem to a simple test case that I can report as a bug...